### ISSUE TITLE
Tests failing due to out of date types

### ISSUE THREAD

**Issue by MisterAJ** [faceyspacey/redux-first-router#319]
I love the package!

Can we get an updates to the types?

```
Test suite failed to run

TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option): src/store/rootReducer.ts:7:43 - error TS2554: Expected 2-3 arguments, but got 1.

7 const { reducer, middleware, enhancer } = connectRoutes(routesMap);
                                            ~~~~~~~~~~~~~~~~~~~~~~~~

      node_modules/@types/redux-first-router/index.d.ts:231:5
        231     routesMap: RoutesMap<TKeys, TState>,
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        An argument for 'routesMap' was not provided.```

**Comment by ScriptedAlchemy** [comment]
Homie, much love for using our work. But those types were done by some random. I didn’t know they existed till someone opened a issue.  I love helping out those who use our work. But I don’t even know where the url is for these types and I’ve never even seen typescript. I think it’s under a monorepo that hold a bunch of types. Troll thru closed issues and you will see this conversation many times. If anyone can PR that author. It would save me an essay and help you guys out.  I’d do it if I could write it. But consider my plateful of work - I am  involved in many projects and other than Rudy, it’s a one man band for last year while faceyspacey is occupied on things other than GitHub repos :) I maintain a lot, I can’t invest the cognitive load of learning some sub/superset of JavaScript. I’d like to. But Rudy https://github.com/respond-framework/rudy 

It Is my absolute core focus. RFR is due to enter long term support - I’ll accept merges and bug fixes. But no features- that’s what Rudy is. 

RFR2 is actually Rudys core, just the base core. I pushed RFR2 to make sure thebwork we did last year (yeah surprise RFR 2 is a year old) 

Pretty much, we gave you a stack of fixes, plus the year old Rudy core. Then forked thatband spent a year build Rudy which was moved like last week to its in Respond. 


TLDR; community wrote them, I don’t even know how to find the TS defs or what TS really is :)

**Comment by ScriptedAlchemy** [comment]
If you like OSS, we are looking for jjjst a little help to finalize one more feature before taking Rudy life. It’s a big one and faceyspacey is blueprinting it now - me and danial have basically been smashing out Rudy under the guidance of faceyspacey- but it’s hard work and every little bit helps us hugely.

**Comment by MisterAJ** [comment]
Understandable. I’ll do what I can to help out! Thanks! :-)
